pkg/observable/channel/replay.go

```diff
@@ -0,0 +1,209 @@
+package channel
+
+import (
+	"context"
+	"log"
+	"sync"
+	"time"
+
+	"pocket/pkg/observable"
+)
+
+// TODO_CONSIDERATION: perhaps this should be parameterized.
+
+// replayPartialBufferTimeout is the duration to wait for the replay buffer to
+// accumulate at least 1 value before returning the accumulated values.
+const replayPartialBufferTimeout = 100 * time.Millisecond
```

Comment:
Do you think we need the word `Partial` in this varname? I was going to suggest `Single` after reading the comment, but feel like removing altogether also works

Reply:
The rationale behind including the word `Partial` was to serve as a reminder that this timeout only applies in cases where the replay buffer is not full. If it is the timeout is ignored.